notifier.go

```diff
@@ -39,6 +39,7 @@ func notifyStateChange(info *monitorInfo, newStatus bool) error {
 	}
 
 	req, err := http.NewRequest(http.MethodPost, info.URL, bytes.NewBuffer(body))
+	req.Header.Set("Content-Type", "application/json; charset=UTF-8")
```

Comment:
I believe the UTF should be lowercase, technically?

```suggestion
	req.Header.Set("Content-Type", "application/json; charset=utf-8")
```

Reply:
Agree its normally lowercase, even though it works as its case-insensitive.  Thanks for the feedback

https://datatracker.ietf.org/doc/html/rfc7230#section-3.2